certified-connectors/RiskIQPassiveTotal/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Token",
+          "description": "The Token for this api",
+          "tooltip": "Please provide the Token",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Secret",
+          "description": "The Secret for this api",
+          "tooltip": "Please provide the Secret",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FF8200",
+    "capabilities": [],
+    "publisher": "RiskIQ"
```

Comment:
`stackOwner` property is missing. Can be the same value as `publisher` in this case

Reply:
Missing properties added in separate [PR](https://github.com/microsoft/PowerPlatformConnectors/pull/724/files)